### DISCUSSION TITLE
Speed limit warning without active navigation

### DISCUSSION THREAD

**Discussion by Wahnsihansi** [osmandapp/OsmAnd#19948] [Ideas]
I really appreciate the speed limit warning when I drive too fast. But it only works when I have an active navigation running. It would be nice to get a speed warning when the app is open but no navigation running.

**Comment by ZeljkoBG** [comment]
Don't worry then you didn't exceed the speed limit.

On what basis should you get a speeding warning if you are not moving?

**Comment by Wahnsihansi** [comment]
Maybe I couldn't find the right words, I am sorry, I am not a native english speaker.
I mean it gives me speed warnings when I enter a destination and start navigation, but it doesn't give me speedwarnings when I drive with the app&map open but no navigation running. Would be nice if that was possible for routes that I know very well, because sometimes I still need a reminder if I am driving too fast.

**Comment by vshcherb** [comment]
**Android** version already has that feature supported for Speedometer widget and Max Speed widget

**Reply by Wahnsihansi** [reply]
You're right, I forgot to say that I want a VOICE warning. Without navigation there is no voice warning possible, it only shows the warning on the screen. I don't need voice navigation advice on routes that I know well obviously, but I'd still appreciate voice speed warnings.

**Reply by vshcherb** [reply]
Yes, voice warning is not implemented as it requires special setting

**Reply by Wahnsihansi** [reply]
Easier than implementing voice warning without active navigation might be making an option to turn off voice navigation without turning off voice speed warning. That way I can just start a navigation when I go to work, still get voice speed warnings but I'm not disturbed by voice navigation.
